tests/ipv6-evasion/ipv6-atomic-fragments-toobig/README.md

```diff
@@ -0,0 +1,7 @@
+# Description
+
+Test an attack causing atomic fragments and therefore a DOS attack as described in RFC8021
+
```

Comment:
I am not sure what is expected from Suricata here.
To have a default rule that triggers on packets found by Wireshark filter `icmpv6.type == 2 && icmpv6.mtu == 1200` ?

Reply:
could be one possibility, yes

Reply:
Done in #187  the right condition is `icmpv6.mtu < 1280`